pom.xml

```diff
@@ -2237,6 +2237,13 @@
           <inputEncoding>${project.build.sourceEncoding}</inputEncoding>
           <outputEncoding>${project.reporting.outputEncoding}</outputEncoding>
         </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>scala_style_checker</artifactId>
```

Comment:
Yikes, do we have to maintain and publish this as a module to use it? it may not be worth it